sway-lib-std/src/bytes.sw

```diff
@@ -416,7 +416,9 @@ impl Bytes {
     /// assert(bytes.is_empty());
     /// ```
     pub fn clear(ref mut self) {
+        self.buf.ptr = alloc_bytes(0);
         self.len = 0;
+        self.buf.cap = 0;
```

Comment:
Should we be doing the same in `Vec.sw`?

Reply:
probably? @jtriley-eth has WIP in Vec now.